modules/Makefile

```diff
@@ -16,8 +16,9 @@ obj-y += cas_disk/
 else
 
 OCFDIR=$(PWD)/../ocf
-KERNEL_DIR ?= "/lib/modules/$(shell uname -r)/build"
 PWD=$(shell pwd)
+KERNEL_VERSION ?= "$(shell uname -r)"
+KERNEL_DIR ?= "/lib/modules/$(KERNEL_VERSION)/build"
```

Comment:
Looks like this breaks building CAS with kernel provided by source path (KERNEL_DIR=<kernel source> make ...). You can keep the old implementation and get the kernel version by "make -C $KERNEL_DIR kernelversion"

Reply:
It doesn't. I intentionally left `?=` for both to make it possible to override either `KERNEL_VERSION` or `KERNEL_DIR`.

Reply:
Given that it's not used anywhere else it doesn't seem to be an issue.

Reply:
It's confusing at least. If KERNEL_VERSION is not used anywhere then I don't see any point in this change